test/manual/tls_support.test.ts

```diff
@@ -114,6 +115,29 @@ describe('TLS Support', function () {
     });
   });
 
+  context('when providing tlsCRLFile', () => {
```

Comment:
We don't have a CRL in drivers-tools that wouldn't reject the CA we have, so just have an integration test for the failure. Looked at the Python driver and they do the same.